app/views/hyrax/base/_work_type.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="work-type-tag">
+  <% if Hyrax.config.display_work_icon? %>
+    <span class="<%= t("hyrax.icons.#{presenter.model}") %>" aria-hidden="true"></span>
```

Comment:
Isn't there a css_icon helper thing to use here?

Reply:
I don't this so, not yet anyway. I was getting an error message with the helper syntax. I'll review adding one.

Reply:
`Hyrax::ModelIcon.css_class_for(presenter.model.class)` is what I'm looking for here.